<!doctype html>
<html lang="en">

<head>
    <title>Bioinformatics Center, CCMB</title>
    <!-- Required meta tags -->
    <meta="utf-8">
        <meta content="width=device-width, initial-scale=1.0" name="viewport" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
        <script type="text/javascript" src="./js/main.js"></script>
        <link href="./css/main.css" rel="stylesheet" />
        <!--     Fonts and icons     -->
        <link rel="preconnect" href="https://fonts.gstatic.com">
        <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap" rel="stylesheet">
        <link rel="stylesheet" type="text/css"
            href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:400,700|Material+Icons" />
      <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500&display=swap" rel="stylesheet">

        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/latest/css/font-awesome.min.css">        
        <script src="https://kit.fontawesome.com/1fe5682f52.js" crossorigin="anonymous"></script>
        <!-- Material Kit CSS -->
        <link href="./material-kit-master/assets/css/material-kit.css" rel="stylesheet" />
        <!--   Core JS Files   -->
        <script src="./material-kit-master/assets/js/core/jquery.min.js" type="text/javascript"></script>
        <script src="./material-kit-master/assets/js/core/popper.min.js" type="text/javascript"></script>
        <script src="./material-kit-master/assets/js/core/bootstrap-material-design.min.js" type="text/javascript">
        </script>
        <script src="./material-kit-master/assets/js/plugins/moment.min.js"></script>
        <script src="./material-kit-master/assets/js/plugins/nouislider.min.js" type="text/javascript"></script>
        <script async defer src="https://buttons.github.io/buttons.js"></script>
        <!-- Control Center for Material Kit: parallax effects, scripts for the example pages etc -->
        <script src="./material-kit-master/assets/js/material-kit.js" type="text/javascript"></script>
        <style>
            ul li {
                list-style-type: none;
                text-align: justify
            }
        </style>
</head>

<body>
    <nav class="navbar navbar-color-on-scroll navbar-transparent fixed-top navbar-expand-lg" color-on-scroll="50">
        <div class="container">
            <div class="navbar-translate">
                <a class="navbar-brand" href="https://www.ccmb.res.in/"><img class="ccmb-logo mb-5"
                        src="./images/CCMB-logo.png" alt="CCMB-LOGO"> </a>
                <button class="navbar-toggler" type="button" data-toggle="collapse" aria-expanded="false"
                    aria-label="Toggle navigation">
                    <span class="sr-only">Toggle navigation</span>
                    <span class="navbar-toggler-icon"></span>
                    <span class="navbar-toggler-icon"></span>
                    <span class="navbar-toggler-icon"></span>
                </button>
            </div>
            <div class="collapse navbar-collapse">
                <ul class="navbar-nav ml-auto" style="color:black;">
                    <li class="nav-item">
                        <a href="./index.html" class="nav-link">
                            HOME
                        </a>
                    </li>
                    <li class="nav-item">
                        <a href="./research.html" class="nav-link">
                            RESEARCH
                        </a>
                    </li>
                    <li class="nav-item">
                        <a href="./tools.html" class="nav-link">
                            TOOLS
                        </a>
                    </li>
                    <li class="nav-item">
                        <a href="./publication.html" class="nav-link">
                            PUBLICATIONS
                        </a>
                    </li>
                    <li class="nav-item">
                        <a href="./team.html" class="nav-link">
                            TEAM
                        </a>
                    </li>
                    <li class="nav-item">
                        <a href="./join.html" class="nav-link">
                            JOIN US
                        </a>
                    </li>
                    <li class="nav-item">
                        <a href="./blog.html" class="nav-link" target="_blank">
                            BLOG
                        </a>
                    </li>
                </ul>
            </div>
        </div>
    </nav>

    <div class="container">
        <div class="empty-div"></div>
    </div>

    <div class="main main-raised main-body">
        <div class="container ">
            <div class="section">
                <h1 class="text-center mb-5" id="cdbest">Tools and Databases</h1>
                <!-- <hr style="width: 70%; text-align: center"> -->
                <div class="row">
                    <div class="col-md-12">cdBEST - Chromatin Domain Boundary Element Search Tool</div>
                    <div class="col-md-6">
                        <div class="research_column">
                            <img class="tool_cdbest" src="./images/tools_cdbest.jpg"
                                alt="cdBEST - Chromatin Domain Boundary Element Search Tool">
                        </div>
                    </div>
                    <div class="col-md-6">
                        <p class="text-justify p-2">Chromatin domain boundary elements prevent inappropriate interaction
                            between distant or closely spaced regulatory elements and restrict enhancers and silencers
                            to correct target promoters. In spite of having such a general role and expected frequent
                            occurrence genome wide, there is no DNA sequence analysis based tool to identify boundary
                            elements. We developed the chromatin domain Boundary Element Search Tool (cdBEST) to
                            identify boundary elements. cdBEST uses known recognition sequences of boundary interacting
                            proteins and looks for ‘motif clusters’. Using cdBEST, we identified boundary sequences
                            across 12 Drosophila species. Of the 4576 boundary sequences identified in Drosophila
                            melanogaster genome, >170 sequences are repetitive in nature and have sequence homology to
                            transposable elements. Analysis of such sequences across 12 Drosophila genomes showed that
                            the occurrence of repetitive sequences in the context of boundaries is a common feature of
                            drosophilids. Enhancer-blocking assays on a subset of the cdBEST boundaries show that 80% of
                            them indeed function as boundaries in vivo. cdBEST thus provides a better understanding of
                            chromatin domain boundaries in Drosophila and sets the stage for comparative analysis of
                            boundaries across closely related species.</p>
                        <div>
                            <a href="https://academic.oup.com/nar/article/40/10/4385/2411561">Publications</a>
                            <a class="pl-3" href="#">Docs</a>
                            <a class="pl-3"
                                href="https://watermark.silverchair.com/gks045.pdf?token=AQECAHi208BE49Ooan9kkhW_Ercy7Dm3ZL_9Cf3qfKAc485ysgAAAp0wggKZBgkqhkiG9w0BBwagggKKMIIChgIBADCCAn8GCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQM0vbLBKyWwIV5Jg37AgEQgIICULcXF0uTRc_DzxA5JIDuDGTAmb7i3yZuTFwP5vTIUBiGSyXrVsxkEfhiCpwNOtW1XuzaSwGV8W3sDdjlpDkmffc6DvKdKwuAQ8bRGE1YZXXYaARpqthQcR0SPiQejYJjSdEgVGhD5ZG8RCm2UhQQqUGrjudEjspgy6zeWbEBFtFJmWdnvRZUlTIiGfFZ20OjaqEr_JoKv82Lfy4x1l19Q392AtgS2nMnOM8oGqg2rzyt2y2xbvgMb_xcblVcMmdu5oZ906fWqaNKHdQD9u2u0LMorfkMQ8F7_azmPsqczBgQh5lDHtGRKKVJt6N_fbYFJTvPHQne175brMz1VE8Eq0j5q2fOOIGloaQ_r4NH7IIvfoPIG2pjlVgNK5eWXdEiKOcjK_Kdk05Gfb-n8Rtg29lZu9ccm6NCADZ72vSed-0Bg75KX2Zyp1W10R76zi_KqRW9DlgL2HcpEf0t4TFt4olfaqCjyCdtJst0odEfpP6JVVb7zqxdqHOcp9ULvj4v9nAyE3F9SsbnJlo4DZAa9B3aLTGQVZ-rVqO_CIBIytLBL5EFPoTSYEuU7B2I1HlIDzpELtLIFD_RTRHhJeChlYfD5gbJRBe-4ahYjx9tsgoslC5GEk88iV6TIEhGqR2QafUbebfny0aHveSJlqMkkRNP3ct6h3UJjXAF1KT4qVcc0XIpQOyPv61tqtBLCsF9lEN7o4jovx2cpmratyt3-8u5WtR4GkWde2Zty_5OVmnqCfVX5_uXh_26MXcle0cd_t0OwTAH2xKhhfQFWV2xnfQ"
                                download>Download</a>
                        </div>
                    </div>
                </div>
                <section id="cstate"></section>
                <br><br>

                <hr>
                <div class="row mt-4">
                    <div class="col-md-12">C-State - 'C' the Chromatin State</div>
                    <div class="col-md-6">
                        <p class="text-justify p-2">Comparative epigenomic analysis across multiple genes presents a
                            bottleneck for bench biologists working with NGS data. Despite the development of
                            standardized peak analysis algorithms, the identification of novel epigenetic patterns and
                            their visualization across gene subsets remains a challenge. We developed a fast and
                            interactive web app, C-State (Chromatin-State), to query and plot chromatin landscapes
                            across multiple loci and cell types. C-State has an interactive, JavaScript-based graphical
                            user interface and runs locally in modern web browsers that are pre-installed on all
                            computers, thus eliminating the need for cumbersome data transfer, pre-processing and prior
                            programming knowledge. C-State is unique in its ability to extract and analyze multi-gene
                            epigenetic information. It allows for powerful GUI-based pattern searching and
                            visualization. Its potential for identifying user-defined epigenetic trends in context of
                            gene expression profiles is demonstrated at the <a
                                href="https://www.ccmb.res.in/index.php?/rakeshmishra/c-state/dmel_het/"
                                target="_blank">Het C-State</a> page and in the case studies in
                            the <a href="https://www.ccmb.res.in/index.php?/rakeshmishra/c-state/">user manual</a> </p>
                        <div>
                            <a
                                href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1786-6">Publication</a>
                            <a class="pl-3" href="https://www.ccmb.res.in/index.php?/rakeshmishra/c-state/">Website</a>
                            <a class="pl-3" href="https://github.com/RKMlab/c-state">Source</a>
                        </div>
                    </div>
                    <div class="col-md-6">
                        <div class="research_column">
                            <img class="tool_Cstate" src="./images/tools_cstate.png"
                                alt="C-State - 'C' the Chromatin State">
                        </div>
                    </div>
                </div>
                <section id="get"></section>
                <br><br>

                <hr>
                <div class="row mt-4">
                    <div class="col-md-12">GET - Genomes Exploration Tool</div>
                    <div class="col-md-6">
                        <div class="research_column">
                            <img class="tool_cdbest" src="./images/tools_get.png"
                                alt="C-State - 'C' the Chromatin State">
                        </div>
                    </div>
                    <div class="col-md-6">
                        <p class="text-justify p-2">The NCBI Genome database is a collection of information on all the
                            genome sequencing projects done and are in progress till date. NCBI provides a very basic
                            browsing interface, which represents the whole data in a tabular format and has limited
                            options for a user to explore through the genomes and find the information they need. To
                            provide a user-friendly and interactive tool to explore this data, we developed Genomes
                            Exploration Tool (GET) using a JavaScript plotting library called d3.js. GET uses data
                            provided by NCBI and converts it into clear, interactive and visually appealing plots, which
                            can be navigated and interacted with as the user desires. Using attributes such as Genome
                            size, GC content, Number of genes and proteins etc. for more than 18000 genomes, users can
                            explore the genomes using bar plots, scatter plots, box-and-whisker plots and histograms.
                        </p>
                        <div>
                            <a href="https://github.com/avvaruakshay/GET">Source</a>
                        </div>
                    </div>
                </div>
                <section id="msdb"></section>
                <br><br>

                <hr>
                <div class="row mt-4">
                    <div class="col-md-12">MSDB - A comprehensive database of microsatellites</div>
                    <div class="col-md-6">
                        <p class="text-justify p-2">Microsatellites, also known as Simple Sequence Repeats (SSRs), are
                            short tandem repeats of 1–6 nt motifs present in all genomes, particularly eukaryotes.
                            Besides their usefulness as genome markers, SSRs have been shown to perform important
                            regulatory functions, and variations in their length at coding regions are linked to several
                            disorders in humans. Microsatellites show a taxon-specific enrichment in eukaryotic genomes,
                            and some may be functional. MSDB (Microsatellite Database) is a collection of >650 million
                            SSRs from 6,893 species including Bacteria, Archaea, Fungi, Plants, and Animals. This
                            database is by far the most exhaustive resource to access and analyze SSR data of multiple
                            species. In addition to exploring data in a customizable tabular format, users can view and
                            compare the data of multiple species simultaneously using our interactive plotting system.
                            MSDB is developed using the Django framework and MySQL.</p>
                        <div>
                            <a href="https://academic.oup.com/nar/article/48/D1/D155/5584837">Publication</a>
                            <a class="pl-3" href="http://data.ccmb.res.in/msdb/">Website</a>
                        </div>
                    </div>
                    <div class="col-md-6">
                        <div class="research_column">
                            <img class="tool_cdbest" src="./images/tools_microsatellites.png"
                                alt="MSDB - A comprehensive database of microsatellites">
                        </div>
                    </div>
                </div>
                <section id="perf"></section>
                <br><br>

                <hr>
                <div class="row mt-4">
                    <div class="col-md-12">PERF - Perfect, Exhaustive Repeat Finder</div>
                    <div class="col-md-6">
                        <div class="research_column">
                            <img class="tool_cdbest" src="./images/tools_perf.png"
                                alt="PERF - Perfect, Exhaustive Repeat Finder">
                        </div>
                    </div>
                    <div class="col-md-6">
                        <p class="text-justify p-2">PERF is a Python package developed for fast and accurate
                            identification of microsatellites (SSRs) from DNA sequences. The existing tools for SSR
                            identification have one or more caveats in terms of speed, comprehensiveness, accuracy,
                            ease-of-use, flexibility and memory usage. PERF was designed to address all these problems.
                        </p>
                        <p class="text-justify p-2">
                            PERF is a recursive acronym that stands for "PERF is an Exhaustive Repeat Finder". It is
                            compatible with both Python 2 (tested on Python 2.7) and 3 (tested on Python 3.5). Its key
                            features are:
                        </p>
                        <ul>
                            <li>Fast run time, despite being a single-threaded application. As an example,
                                identification of all SSRs from the entire human genome takes less than 7 minutes. The
                                speed can be further improved ~3 to 4 fold using PyPy (human genome finishes in less
                                than 2 minutes using PyPy v5.8.0)</li>
                            <li>Linear time and space complexity (O(n))</li>
                            <li>Identifies perfect SSRs</li>
                            <li>100% accurate and comprehensive - Does not miss any repeats or does not pick any
                                incorrect ones</li>
                            <li>Easy to use - The only required argument is the input DNA sequence in FASTA format</li>
                            <li>Flexible - Most of the parameters are customizable by the user at runtime</li>
                            <li>Repeat cutoffs can be specified either in terms of the total repeat length or in terms
                                of number of repeating units</li>
                            <li>TSV output and HTML report. The default output is an easily parseable and exportable
                                tab-separated format. Optionally, PERF also generates an interactive HTML report that
                                depicts trends in repeat data as concise charts and tables</li>
                        </ul>

                        <div>
                            <a class="pl-3"
                                href="https://academic.oup.com/bioinformatics/article/34/6/943/4600186">Publication</a>
                            <a class="pl-3" href="https://github.com/rkmlab/perf">Docs</a>
                            <a class="pl-3" href="https://github.com/rkmlab/perf">Source</a>
                        </div>
                    </div>
                </div>
                <br><br>
                <hr>
                <div class="row mt-4">
                    <div class="col-md-12">GEAR-19 - Genome Evolution Analysis Resource for COVID-19</div>
                    <div class="col-md-6">
                        <p class="text-justify p-2">Gear19 database is developed using Node.js framework, vue.js and
                            mongoDB to track the genomic evolution of SARS-CoV-2 virus which is resposible for
                            worldwide pandemic of COVID-19. Gear19 is carrying 3449 indian SARS-CoV-2 sequenced genome
                            data, 4638 variants catalogued and alongwith 9 different clades. In gear19, exploring data
                            in a customizable table format so that users can easily explore and compare the data using
                            interactive and demographic plots. </p>
                        <div>
                            <!-- <a href="https://academic.oup.com/nar/article/48/D1/D155/5584837">Publication</a> -->
                            <a class="pl-3" href="http://data.ccmb.res.in/gear19/">Website</a>
                        </div>
                    </div>
                    <div class="col-md-6">
                        <div class="research_column">
                            <img class="tool_cdbest" src="./images/gear19.png"
                                alt="Gear19 - Genome Evolution Analysis Resource for COVID-19">
                        </div>
                    </div>
                </div>
                <br><br>
                <hr>
                <h2>Other Tools</h2>
                <h4><a href="http://e-portal.ccmb.res.in/e-space/rakeshmishra/risci-tool.html" target="_blank">RISCI</a>
                </h4>
                <h4><a href="http://e-portal.ccmb.res.in/e-space/rakeshmishra/snsalign-tool.html"
                        target="_blank">Sns-Align</a></h4>

            </div>

        </div>
        <br><br><br>

    </div>
    </div>
    <footer class="footer footer-default">
        <div class="container">
            <div class="row">
                <div class="col-md-4">
                    <nav class="float-left">
                        <ul>
                            <li>
                                <a href="https://www.ccmb.res.in/">
                                    CSIR-CCMB
                                </a>
                            </li>
                            <!-- <li>
                                <a href="http://e-portal.ccmb.res.in/e-space/rakeshmishra/home.html">
                                    Lab of Rakesh Mishra
                                </a>
                            </li> -->
                        </ul>
                    </nav>
                </div>

                <div class="col-md-4">
                    <div class="icon">
                        <a class="footer-icons" href="https://github.com/RKMlab">
                            <i class="fab fa-github"></i>
                        </a>
                        <a class="footer-icons" href="https://twitter.com/bic_ccmb">
                            <i class="fab fa-twitter"></i>
                        </a>
                    </div>
                </div>
                <div class="col-md-4">
                    <div class="copyright float-right">
                        &copy;
                        <script>
                            document.write(new Date().getFullYear())
                        </script>,
                        <a href="./index.html">CCMB Genome Research and Integrated Data sciences</a>
                    </div>
                </div>
            </div>
        </div>
    </footer>
</body>

</html>